pkg/observability/observability_config_test.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2020 The Knative Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package observation
+
+import (
+	"testing"
+
+	"knative.dev/pkg/metrics"
+
+	. "knative.dev/pkg/configmap/testing"
+	_ "knative.dev/pkg/system/testing"
+)
+
+func TestNewConfigMap(t *testing.T) {
+	cm, example := ConfigMapsFromTestFile(t, metrics.ConfigMapName())
+
+	if cfg, err := metrics.NewObservabilityConfigFromConfigMap(cm); err != nil {
+		t.Errorf("Expected no errors. got: %v", err)
+	} else if cfg == nil {
+		t.Errorf("NewConfigFromConfigMap(actual) = %v, want non-nil", cfg)
```

Comment:
nit:
```suggestion
		t.Errorf("NewObservabilityConfigFromConfigMap(actual) = %v, want non-nil", cfg)
```

(same below as well)